### PR TITLE
Create ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57128](https://cumulocity.atlassian.net/browse/MTM-57128)
Original PR: [6180](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6180)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Disable dashboard availability select if user has no permission to request roles to populate it. (#6173) [GRAFT][release/y2024] (#6180)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: disable dashboard availability select for users without permission to request roles
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57128
+version: 1018.503.78
+---
+Disable dashboard availability select if user has no permission to request roles to populate it. (#6173) [GRAFT][release/y2024] (#6180)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md:17]: The Claude AI suggested to use the following body:
```suggestion
In the past, the dashboard availability select was always enabled for users, even if they did not have permission to request the roles necessary to populate it. With this change, the dashboard availability select is now disabled for users who do not have permission to request roles. This improves the user experience by avoiding confusion and preventing users from attempting to make changes they are not authorized to make.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Review by jdreSAG — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md (hunk 1 above)
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md:3]:
```suggestion
title: Hide dashboard availability select if user has no permission to request roles to populate it.
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md (hunk 2 above)
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md:17]:
```suggestion
Previously, the dashboard availability select was always enabled for users, even if they did not have permission to request the roles necessary to display availability select list. With this change, the dashboard availability select is now hidden for users who do not have permission to request roles. User need User management READ permission to see and modify availability property of dashboard.
```

**Comment by jdreSAG** [comment]
> @jdreSAG Why do you not use the AI suggestions? The current description is too short, missing the context and the user impact. It should not simply repeat the title. And the title also does not state what is the improvement for the user. The content in brackets should not show up.

I agree, on second thoguht- it needs more context. Added more info as AI suggestion was not precise.

File: content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57128
+version: 1018.503.78
+---
+Previously, the dashboard availability select was always enabled for users, even if they did not have permission to request the roles necessary to display availability select list. With this change, the dashboard availability select is now hidden for users who do not have permission to request roles. User need User management READ permission to see and modify availability property of dashboard.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md:17]:
```suggestion
Previously, the dashboard availability selection was always visible for users, even if they did not have the permission to see it. With this change, the dashboard availability selection is hidden for users who do not have the required permission. Users must have User Management READ permission to see and modify the availability property of a dashboard.
```

File: content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Hide dashboard availability select if user has no permission to request roles to populate it.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1018-503-78-disable-dashboard-availability-select-if-user-has-no-permission-to-request.md:3]:
```suggestion
title: Dashboard availability selection is hidden for users who do not have the required permission
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**